doc/Packet-action-DONOTDROP.md

```diff
@@ -0,0 +1,63 @@
+# SAI_PACKET_ACTION_DONOTDROP packet action
+-------------------------------------------------------------------------------
+ Title       | SAI_PACKET_ACTION_DONOTDROP packet action
+-------------|-----------------------------------------------------------------
+ Authors     | Ashish Singh, Google LLC
+ Status      | In review
+ Type        | Standards track
+ Created     | 01/25/2022
+ SAI-Version | 1.8.3
+
+
+-------------------------------------------------------------------------------
+
+This spec discusses the SAI_PACKET_ACTION_DONOTDROP packet action use-cases.
+
+Goal of this packet action attribute (SAI_PACKET_ACTION_DONOTDROP) is to resolve the conflicts of ACL actions when a packet hits more than one ACLs such that when this packet action is used with an ACL, it serves as an action that prevents the possible drop from another ACL that is in a lower priority table.
+
+This new packet action attribute is in line with the current SAI architecture - actions in higher priority groups take precedence. If an ACL with DROP action is in a higher priority group than this ACL then, DROP action is expected to be honored. Few cases are explained with examples in next section.
+
+# Usage Examples
+### Case 1
+
+| ACL in table T1                     | ACL in table T2                       |
+|-------------------------------------|---------------------------------------|
+| Table priority group:  P1           | Table priority group:  P2             |
```

Comment:
Can you please elaborate on "Table priority group"? Currently, there is no priority attribute for ACL Group table in SAI.

Reply:
Yes. It is good to clarify if you meant SAI_ACL_TABLE_GROUP_MEMBER_ATTR_PRIORITY.

Reply:
But SAI_ACL_TABLE_GROUP_MEMBER_ATTR_PRIORITY is applicable only for the sequential ACL groups. For a sequential ACL group, only the highest priority table is applied so if the highest priority table has a redirect action, it will apply regardless of any conflicting actions in the lower priority tables. So, I'm bit unclear how this new packet action type will be used.

Reply:
done. Thank you.